### ISSUE TITLE
Histogram sound should not stop when adjusting zoomProperty

### ISSUE THREAD

**Issue by matthew-blackman** [phetsims/projectile-data-lab#166]
Zooming in and out of the histogram should not stop the histogram sound sequence. This is coming from updateHistogram being called in the histogram.zoomProperty.link on HistogramNode. Let's find a way to tell the HistogramSonifier not to stop the sound in this case.

**Comment by samreid** [comment]
Proposed fix in the commit. I analyzed all the calls to updateHistogram and saw one other that likely shouldn't cancel the sonification. Ready for review.

**Comment by matthew-blackman** [comment]
This is fixed by the above commit. Reviewing this with @samreid, we opted not to stop the histogram sound when navigating data or samples. That work snuck in to #169 in [this commit](https://github.com/phetsims/projectile-data-lab/commit/378dc39a429bca47ab71173dd58fded672599d37). Sorry about that!

The behavior is correct on all screens, and the code has been reviewed and looks good. Nice work! Closing.
